challenges/src/test/java/challenges/BinarySearchTest.java

```diff
@@ -0,0 +1,47 @@
+
+package challenges;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class BinarySearchTest {
+
+    @Test public void testBinaryStuff() {
+
+        int[] arr = {1, 2, 3, 4, 5, 6};
+        int key = 1;
+
+        int actualOutput = BinarySearch.BinarySearcher(arr, key);
+
+        System.out.println("actualOutput = " + actualOutput);
+        
```

Comment:
it looks like you are trying to find out what to test here. It is correctly outputting that the position of value 1 is 0, your test can check assertEquals(0, actualOutput)

Reply:
Thank you for pointing that out, super bone-head mistake on my part.